bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/model/yaml/YamlModelRepository.java

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.semantics.model.yaml;
```

Comment:
Of course, all the classes in that package have to be moved in another bundle.
WDYT about bundle org.openhab.core.model.core in new package org.openhab.core.model.core.internal.yaml ?
Or is it better to create a new bundle org.openhab.core.model.yaml ?

Reply:
I moved the "common" files to the bundle org.openhab.core.model.core

Reply:
Finally everything was moved in a new bundle org.openhab.core.model.yaml